pkg/splunk/enterprise/configuration.go

```diff
@@ -642,6 +669,20 @@ func updateSplunkPodTemplateWithConfig(client splcommon.ControllerClient, podTem
 	if spec.Defaults != "" {
 		splunkDefaults = fmt.Sprintf("%s,%s", "/mnt/splunk-defaults/default.yml", splunkDefaults)
 	}
+	if appListingConfigMapExists {
+		var appListingFiles []string
+		for key := range appListingConfigMap.Data {
+			if key != configToken {
+				appListingFiles = append(appListingFiles, key)
+			}
+		}
+		// Always sort the slice, so that map entries are ordered, to avoid pod resets
+		sort.Strings(appListingFiles)
+
+		for _, fileName := range appListingFiles {
+			splunkDefaults = fmt.Sprintf("%s%s,%s", "/mnt/app-listing/", fileName, splunkDefaults)
```

Comment:
Since the Deployer and the SHC Member have the same CR, does this get added to the defaults for the SHC member as well?